pset-iterators.js

```diff
@@ -43,6 +43,12 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const firstWhigPresident = presidents.find((president) => {
+	return president.party === 'Whig'
+})
+
+console.log(`The First Whig Party President is ${firstWhigPresident.president}.`)
+
```

Comment:
@cswebdesign Nice job here on this solution using the find method to look up the first president in the whig party.
One recommendation checks your indentation, you currently indented by 4 spaces but you should indent by 2 spaces.
Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L35)